index.js

```diff
@@ -86,7 +86,12 @@ class Auth {
       };
 
       if (config.credentials) {
-        googleAuthClient.fromJSON(config.credentials, addScope);
```

Comment:
@ofrobots previously, I *think* the callback to `fromJSON()` would be executed with:

- err
- auth client instance
- project ID (if detected)

Now that the callback is gone, and the auth client is returned directly from `fromJSON()`, is the project ID being parsed from the provided object and stored somewhere?

Reply:
In the source for `fromJSON` from 0.12.0, it seems like the callback was never called with the project ID: https://github.com/google/google-auth-library-nodejs/blob/v0.12.0/ts/lib/auth/googleauth.ts#L391

I think the only project ID involved is the maybe-`client.projectId` field.

Reply:
Alright, thanks for checking that out!